post-build/action.yml

```diff
@@ -57,9 +57,9 @@ runs:
     - name: Count bottles
       id: bottles
       if: always()
-      uses: Homebrew/actions/count-bottles@master
-      with:
-        working-directory: ${{ inputs.bottles-directory }}
```

Comment:
this caused some issue

```
Error: Homebrew/actions/master/post-build/action.yml: (Line: 60, Col: 57, Idx: 1765) - (Line: 60, Col: 78, Idx: 1786): While parsing a block mapping, did not find expected key.
Error: System.ArgumentException: Unexpected type '' encountered while reading 'action manifest root'. The type 'MappingToken' was expected.
   at GitHub.DistributedTask.ObjectTemplating.Tokens.TemplateTokenExtensions.AssertMapping(TemplateToken value, String objectDescription)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
Error: Failed to load Homebrew/actions/master/post-build/action.yml
```
https://github.com/Homebrew/homebrew-core/actions/runs/8577932249/job/23511213925?pr=168176